docs/arches_flags.json

```diff
@@ -1,859 +1,5144 @@
 [
```

Comment:
Can we ignore this file?

Reply:
Yes, we can even delete it :-)

Reply:
https://github.com/eic/EICrecon/issues/1376
Would be nice to keep the feature going, though. We have the artifacts, but those are not available without login and you can't get a permalink for them.

Reply:
Ignored in 882dc7df